### PR TITLE
Feature/homepage layout

### PR DESCRIPTION
**Pull request by thomasridd**
### What

Added layout for homepage

I have taken and edited the html from the existing website and refactored it for use with wagtail. This has used the prettified version of raw css from the website. Suggest making accessibility tweaks after we get full access to scss files. 

### How to review

Image below to act as preview

image

### CODE REVIEW

File: CMS/templates/base.html [hunk 1]
```diff
@@ -20,6 +20,7 @@
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
     <link href="{% sass_src 'scss/global.scss' %}" rel="stylesheet" type="text/css" >
+    <link href="{% static 'css/crc.css' %}" rel="stylesheet" type="text/css" >
```
Comment (mwnicks) [inline CMS/templates/base.html]: Have you moved the file from scss to css?
The css files are git ignored, so the styling is not in the repo

Comment (thomasridd) [reply CMS/templates/base.html]: yes I have - something is going on with that css and it won't compile as scss

Comment (thomasridd) [reply CMS/templates/base.html]: have updated that with an ignore line in the .gitignore

suggest we move to scss as soon as it becomes an option

Comment (mwnicks) [reply CMS/templates/base.html]: there are a couple of what seem to be typos in the css that scss doesn't like but the css doesn't care. This is fine for now.

**Pull request merged**